posthog/api/error_tracking.py

```
@@ -415,7 +417,7 @@ def update(self, request, *args, **kwargs) -> Response:
         if json_filters:
             parsed_filters = PropertyGroupFilterValue(**json_filters)
             assignment_rule.filters = json_filters
-            assignment_rule.bytecode = self.generate_byte_code(parsed_filters)
+            assignment_rule.bytecode = generate_byte_code(self.team, parsed_filters)
 
         if assignee:
```

One for myself but this should probably throw an error if not present

---

I think it's fine to do an update that doesn't change the rule, just the assignee?